toolchain/gdb.py

```diff
@@ -30,7 +30,7 @@ def setUp(self):
         sm = SoftwareManager()
         dist = distro.detect()
         packages = ['gcc', 'dejagnu', 'flex',
-                    'bison', 'texinfo', 'make', 'makeinfo']
+                    'bison', 'texinfo', 'make']
```

Comment:
@preetithakur is this package not needed on all Distro

Reply:
For RHEL

Reply:
@preetithakur @PraveenPenguin I did not understand here.
if we need to remove only for RHEL and needed for other Distro flavors then this is not the right place to remove the package name and we need to add logic to remove only for specific distro and even distro version if needed. it is ok if it is not needed for all the distros. 
hope you got my view here.

Reply:
@Naresh-ibm i think @preetithakur about to send patch only in RHEL case